### PR TITLE
[Introduction to 3D] Spatial node replaced with Node3D

### PR DESCRIPTION
**Pull request by TheYellowArchitect**
Spatial node does not exist in 4.0
This has been confirmed in https://github.com/godotengine/godot-docs/issues/6241 and fixes it.

There may be other instances of the Spatial node in the docs, but this is the main one and the easiest to stumble (literally first thing in 3D section lol)

### CODE REVIEW

**Review by lucashilles — APPROVED** [review]
LGTM

File: tutorials/3d/introduction_to_3d.rst [hunk 1]
```diff
@@ -16,19 +16,19 @@ In 3D, math is a little more complex than in 2D, so also checking the
 developers, not mathematicians or engineers) will help pave the way for you
 to develop 3D games efficiently.
 
-Spatial node
+Node3D
 ~~~~~~~~~~~~
```
Comment (Calinou) [inline tutorials/3d/introduction_to_3d.rst:20]: Title underline length must match the text (this is a reStructuredText requirement):

```suggestion
Node3D
~~~~~~
```

**Review by Calinou — CHANGES_REQUESTED** [review]
The `img/tuto_3d1.png` screenshot must be updated before this can be merged.

**Comment by TheYellowArchitect** [comment]
I agree. Give me a few days as I suddenly got a small project I must rush to finish, then I will update this with a new image and the ~~~~~~~ suggestion :+1:

**Comment by TheYellowArchitect** [comment]
https://docs.godotengine.org/en/latest/tutorials/3d/introduction_to_3d.html?highlight=spatial#spatial-node
It seems there are 2 more images in the above page using the Spatial node, so I updated them as well.

However, there is a problem with size. My new `tuto_3d2`:
`img/tuto_3d2.png` is 16KB
`img/tuto_3d2.webp` is 6KB

Its over 50% difference, but its just 16KB so no big deal.

However, `tuto_3d3` is a fullscreen image pretty much. The initial snapshot png is 329KB. Exporting it as .png via GIMP makes it 283KB. Via jpg in an acceptable quality, 91 KB. webp in slightly worse quality than jpg is 42KB. For context, existing/current `tuto_3d3.png` is 69KB. I uploaded the .webp, and want to strongly support implementation for .webp (if it doesn't exist already, cant build docs to confirm for myself)
Related to https://github.com/godotengine/godot-docs/issues/6173

Anyway, for `tuto_3d2` I used .png (just 16KB), and for `tuto_3d3` I used .jpg (91KB) instead of .webp (42KB), so there should be no problem merging this. But for when .webp is added, I have also included the `.webp` of the above images in the folder, so they can be used instead with just a simple line change in text, without uploading any image.

P.S. I use `maim` for taking images of editor, and then export them with GIMP

**Comment by Vennnot** [comment]
This can be merged now, no?

**Comment by mhilbrunner** [comment]
This has been fixed in https://github.com/godotengine/godot-docs/pull/6315.
If the images still need to be updated, a new PR is welcome for that :)

**Pull request closed without merging**